pkg/domain/entities/engine_container.go

```diff
@@ -72,6 +72,7 @@ type ContainerEngine interface {
 	PodExists(ctx context.Context, nameOrID string) (*BoolReport, error)
 	PodInspect(ctx context.Context, options PodInspectOptions) (*PodInspectReport, error)
 	PodKill(ctx context.Context, namesOrIds []string, options PodKillOptions) ([]*PodKillReport, error)
+	PodLogs(ctx context.Context, containers string, options PodLogsOptions) error
```

Comment:
Why containers? It seems more sensible to have `pod string` to choose what pod to attach to, and the specific container in `options` (if one is chosen)